README.md

```diff
@@ -1,6 +1,8 @@
-# rofi-mode
+# rofi-mode &middot; [![crates.io](https://img.shields.io/crates/v/cargo-rdme.svg)](https://crates.io/crates/cargo-rdme) [![License](https://img.shields.io/badge/License-MIT-green.svg)](https://github.com/SabrinaJewson/rofi-mode.rs/blob/main/LICENSE)
```

Comment:
Can’t we just have the badges in both the README and crate docs, but put them beneath the tagline? I think that’d be simpler as well as more useful (and we wouldn’t need cargo-rdme).

Reply:
Yeah makes sense, wasn't sure if it was ok to put stuff like that in `lib.rs` (no experience with this apologies)